### ISSUE TITLE
Validate "Start Mission" button isn't working with screen readers / voice control

### ISSUE THREAD

**Issue by misaugstad** [ProjectSidewalk/SidewalkWebpage#3743]
##### Brief description of problem/feature
Below is some feedback that we got from one user that uses voice controls to do validations (edits to fix voice-to-text issues):
>I use [MacOS] and [it's] voice control software. I also use head control. If you ever computer with [MacOS] you can go into settings and then to accessibility ... Turn on voice control. Numbers will be on the screen that you can use to activate commands. When you go into validation the yellow button on the lower right that says [start mission] does not have a number if you are in safari. Other buttons have the numbers. As such, I need to use the head to move my mouse over the start mission and use the voice command "click " to go into the validation. This can be a challenge when I am fatigued and I lack the neck muscles to adjust. If this could be modified so that a number could be used to activate it, it would make it easier.

##### Potential solution(s)
There are various browser tools that can be used to check for accessibility of a site. We can use one of those to take a look at the mission start screen. They will usually offer suggestions on what's missing! First, verify that you're seeing that something is missing. Hopefully this is consistent across other OS's and browsers. Then fix. If whoever fixes this isn't on a Mac, I'll do a final test myself on my Macbook before merging the PR!
